components/authorize/authorize.tsx

```diff
@@ -9,7 +9,7 @@ import {OAuthApp} from '@mattermost/types/integrations';
 import icon50 from 'images/icon50x50.png';
 import FormError from 'components/form_error';
 import {browserHistory} from 'utils/browser_history';
-import FormattedMarkdownMessage from 'components/formatted_markdown_message.jsx';
+import FormattedMarkdownMessage from 'components/formatted_markdown_message';
 
 export type Params = {
     responseType: string | null;
```

Comment:
In L87
can we do something about it and open a seperate ticket
<img width="956" alt="image" src="https://user-images.githubusercontent.com/17708702/186593123-a2ce5332-f375-4346-98cd-cbb9bc62c882.png">